src/plugins/PasteHandler.ts

```diff
@@ -34,11 +34,11 @@ export default class PasteHandler extends Extension {
     return [
       new Plugin({
         props: {
-          transformPastedHTML: (html: string) => {
-            // TODO: this is a temp fix for pasting headers..
-            //  uncomment to see the error :) this is to be fixed ASAP
-            return replaceHeaderByStrong(html);
-          },
+          // transformPastedHTML: (html: string) => {
+          //   // TODO: this is a temp fix for pasting headers..
+          //   // uncomment to see the error :) this is to be fixed ASAP
+          //   return replaceHeaderByStrong(html);
+          // },
```

Comment:
btw, in most cases, all commented code should be removed! If needed later, we can rely on git to get it back

Reply:
got it will keep in mind for future